sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinTreeVisitor.java

```diff
@@ -273,7 +276,17 @@ private Tree createMatchTree(TreeMetaData metaData, KtWhenExpression element) {
       subjectExpression,
       whenExpressions.stream()
         .map(MatchCaseTree.class::cast)
-        .collect(Collectors.toList()));
+        .collect(Collectors.toList()),
+      getWhenKeyword(element));
+  }
+
+  @NotNull
```

Comment:
This annotation is not necessary as by default we have `@javax.annotation.ParametersAreNonnullByDefault` on all packages. So a warning would be raised even without it